### PR TITLE
Slightly tweak the "no jar files" errors (lp: #1606352)

### PR DESCRIPTION
**Pull request by seb128**
https://bugs.launchpad.net/snapcraft/+bug/1606352

### CODE REVIEW

**Comment by snappy-m-o** [comment]
Can one of the admins verify this patch?

**Comment by snappy-m-o** [comment]
Can one of the admins verify this patch?

File: snapcraft/plugins/ant.py [hunk 1]
```diff
@@ -47,7 +47,7 @@ def build(self):
         self.run(['ant'])
         files = glob.glob(os.path.join(self.builddir, 'target', '*.jar'))
         if not files:
-            raise RuntimeError('could not find any built jar files for part')
+            raise RuntimeError('Could not find any built jar files for part')
```
Comment (sergiusens) [inline snapcraft/plugins/ant.py]: maybe even better and make it
```python
raise RuntimeError('Could not find any built jar files for part {!r}'.format(self.name))
```

Comment (sergiusens) [inline snapcraft/plugins/ant.py]: same for all others.

**Comment by sergiusens** [comment]
Thanks for the improvement to the plugin, I've asked for a minor addition, feel free to say no as this contribution is already good

**Pull request merged**